### PR TITLE
Azure Virtual Machine Scale Sets closes #198

### LINKED ISSUE #198: Azure - describe products available for Compute such as Virtual Machine Scale Sets

### PR DESCRIPTION
**Pull request by jordan-grindrod**

### CODE REVIEW

**Comment by tvaidotas** [comment]
Why is this PR modifying like 21 files? doesn't make sense

**Comment by jordan-grindrod** [comment]
looks like circleci did something weird here - lots of commits made before i even picked it up?

File: topics/azure/modules/vm-scale-sets/README.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+Doing this increases the resiliency of the application, as it is able to easily handle surges in traffic. 
+
+This also allows for an update, or maintenance, to be performed on one application instance (one VM in the scale set), whilst traffic is directed to other instances. 
+
+Doing this means your application will still be available during updates and maintenance windows.
+
+Other benefits include:
+
+* Easy to create and manage multiple, consistant VMs
```
Comment (LukeBenson) [inline topics/azure/modules/vm-scale-sets/README.md]:
```suggestion
* Easy to create and manage multiple, consistent VMs
```

File: topics/azure/modules/vm-scale-sets/README.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+The first thing we need to do is create a **Load Balancer**:
+
+1. In the search box at the top of the home page, we can search "Load Balancer" and click on it when it pops up
+
+2. We'll click **+Add** in the top left
+
+3. In the **Basics** tab of the **Create a load balancer** page, we can enter or select the following:
+
+* Subscription - our sunscription
```
Comment (LukeBenson) [inline topics/azure/modules/vm-scale-sets/README.md]:
```suggestion
* Subscription - our subscription
```

**Pull request merged**